### PR TITLE
Crm

### PR DESCRIPTION
**Pull request by deepak-negi-web**
CRM Pull request

### CODE REVIEW

**Comment by prvnbist** [comment]
@deepak-negi-web move all the icons to root shared assets folder

**Comment by prvnbist** [comment]
All the utils should at the root folder, for eg; `/apps/crm/utils`

File: src/apps/crm/components/ContactInfoCard/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/destructuring-assignment */
```
[the remark this answers is not in the document]
Comment (deepak-negi-web) [reply src/apps/crm/components/ContactInfoCard/index.js]: Changes are made from my side. Kindly Re-review it.

**Comment by deepak-negi-web** [comment]
I have done the changes please re-review the code

**Pull request merged**